cmd/tempo-vulture/main.go

```diff
@@ -141,19 +142,53 @@ func main() {
 			metricTracesErrors.WithLabelValues("requestfailed").Add(float64(metrics.requestFailed))
 			metricTracesErrors.WithLabelValues("notfound").Add(float64(metrics.notFound))
 			metricTracesErrors.WithLabelValues("missingspans").Add(float64(metrics.missingSpans))
+			metricTracesErrors.WithLabelValues("incorrectresult").Add(float64(metrics.incorrectResult))
 		}
 	}()
 
 	http.Handle(prometheusPath, promhttp.Handler())
 	log.Fatal(http.ListenAndServe(prometheusListenAddress, nil))
 }
 
+func intervalsBetween(start, stop time.Time, interval time.Duration, retention time.Duration) []time.Time {
+	if stop.Before(start) {
+		return nil
+	}
+
+	intervals := []time.Time{start}
+	next := start.Round(interval)
```

Comment:
time.Round will round down for the first half of the interval.  I think this means that it can choose a `next` that is before `start`.  Thoughts?

Reply:
Ah yes, I've just updated to include only time windows after the start.  How's that look to you now?